frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -491,6 +492,12 @@ export const eventUsageLogic = kea<eventUsageLogicType>([
         reportOnboardingProductSelected: (productKey: string) => ({ productKey }),
         reportOnboardingCompleted: (productKey: string) => ({ productKey }),
         reportSubscribedDuringOnboarding: (productKey: string) => ({ productKey }),
+        // command bar
+        reportCommandBarStatusChanged: (status: BarStatus) => ({ status }),
+        reportCommandBarSearch: (queryLength: number) => ({ queryLength }),
+        reportCommandBarSearchResultOpened: (type: ResultType) => ({ type }),
+        reportCommandBarActionSearch: (query: string) => ({ query }),
```

Comment:
I figured logging the query here is okay, as this is something like "Toggle dark mode", "Create a dashboard", etc. Happy to change this to query length like for the search however

Reply:
It's probably fine. Ideally we would have a generic helper that could try its best to filter out anything sensitive like IDs or emails (and then use that in the recorder as well) but the risk here is low